middleware/auth.js

```diff
@@ -59,9 +57,12 @@ const ensureRights = (req, res, next) => {
     next();
   } else {
     req.session.destroy(() => {
+      let url = '/admin/login'
```

Comment:
Is there a reason you don't create the complete url in one go? I see you do it in the rest of the code:

```
let url = `${process.env.APP_URL}/admin/users
```

Even better would be to use a function to create the absolute url (that correctly handles url concatenation). To mitigate problems with leading/trailing/absent '/' in the APP_URL or path.

Reply:
Yes you are right, forgot these two.

About the function to create a clean absolute URL you are also right, but will do this later.